### PR TITLE
Lab 4

### PR DESCRIPTION
**Pull request by AndrewPerel**

### CODE REVIEW

File: Lab 4/Lab 4.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function main()
+{
+    const textareaROSTER = document.getElementById("roster");
+
+    textareaROSTER.value += "FirstName LastName\n";
+    textareaROSTER.value += "------------------\n";
+    textareaROSTER.value += document.getElementById("firstName1").value + " " + document.getElementById("lastName1").value + "\n";
```
Comment (akennis) [inline Lab 4/Lab 4.js:7]: Would be good to use a loop here and also interpolation to form the strings.

**Review by akennis — APPROVED** [review]
Good job - putting you down for a 93% - let me know if you address my comment and I'll review this again.  Thanks.

File: Lab 4/Lab 4.js (hunk 1 above)
Comment (AndrewPerel) [reply Lab 4/Lab 4.js]: Updated the code with a for loop and interpolation; a lot fewer lines now

File: Lab 4/Lab 4.js [hunk 2]
```diff
@@ -2,11 +2,10 @@ function main()
...
-    textareaROSTER.value += document.getElementById("firstName1").value + " " + document.getElementById("lastName1").value + "\n";
-    textareaROSTER.value += document.getElementById("firstName2").value + " " + document.getElementById("lastName2").value + "\n";
-    textareaROSTER.value += document.getElementById("firstName3").value + " " + document.getElementById("lastName3").value + "\n";
-    textareaROSTER.value += document.getElementById("firstName4").value + " " + document.getElementById("lastName4").value + "\n";
-    textareaROSTER.value += document.getElementById("firstName5").value + " " + document.getElementById("lastName5").value + "\n";
+    textareaROSTER.value += `FirstName LastName
+------------------\n`
+
+    for (let n = 1; n < 6; n++) {
+        textareaROSTER.value += document.getElementById("firstName" + String(n)).value + " " + document.getElementById("lastName" + String(n)).value + "\n";
```
Comment (akennis) [inline Lab 4/Lab 4.js:9]: Thanks for fixing this!

**Review by akennis — APPROVED** [review]
Looks good now!

**Pull request merged**